TL improvement (#380)
07th-mod/higurashi-console-arcs: Update/omot_024.txt

//r母を失ってからは、…その対象は兄へと移った。叔父夫婦が憎いというよりも、兄が自分を守るために構ってもらえることが、嬉しかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "母を失ってからは、…その対象は兄へと移った。叔父夫婦が憎いというよりも、兄が自分を守るために構ってもらえることが、嬉しかった。",

> 			NULL, "And after losing my mother... the target moved to my brother. Rather than hate my aunt and uncle, I was glad my brother cared me to protect me.", Line_Normal);

Reviewer: ```suggestion
			NULL, "And after losing my mother... the target moved to my brother. Rather than hate my aunt and uncle, I was glad my brother cared enough to protect me.", Line_Normal);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f6d9a8>鷹野</color>", NULL, "<color=#f6d9a8>Takano</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 9, "ps3/s14/09/vt4bc_takano1029", 256, TRUE);
		OutputLine(NULL, "「そして、私を支援するって話も…雛見沢症候群の研究を継続させるって甘い誘いも、全部なかったことになった！！」",
				NULL, "\"And they deserted all about supporting me... They canceled their heartfelt invitation to continue my Hinamizawa Syndrome research!!\"", Line_Normal);
		ClearMessage();

	//鷹野rvS14/09/VT4Bc_takano1030.「私がわざわざ大がかりなことをしなくても、勝手に権力が転がり込んできたんだから！！kvS14/09/VT4Bc_takano1031.　つまり…私を焚きつけてきたやつらは、最初から勢力争いが目的だったってわけよ！！」
		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f6d9a8>鷹野</color>", NULL, "<color=#f6d9a8>Takano</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 9, "ps3/s14/09/vt4bc_takano1030", 256, TRUE);
		OutputLine(NULL, "「私がわざわざ大がかりなことをしなくても、勝手に権力が転がり込んできたんだから！！",

> 				NULL, "\"Because before I cause a major incident, they were already able to seize power with little effort!!", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, "\"Because even before I caused a major incident, they were already able to seize power with little effort!!", Line_WaitForInput);
```

		ModPlayVoiceLS(4, 9, "ps3/s14/09/vt4bc_takano1031", 256, TRUE);
		OutputLine(NULL, "　つまり…私を焚きつけてきたやつらは、最初から勢力争いが目的だったってわけよ！！」",

> 				NULL, " In other words... the people who instigated me were aiming to fight over their internal domination from the start!!\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " In other words... the people instigating me were fighting a power struggle from the start!!\"", Line_Normal);
```
